onnxruntime/python/tools/transformers/models/llama/convert_to_onnx.py

```diff
@@ -0,0 +1,474 @@
+import argparse
+import logging
+import os
+import sys
+import tempfile
+from itertools import chain
+from typing import List
+
+import onnx
+import torch
+from transformers import LlamaConfig, LlamaForCausalLM
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", ".."))
+from benchmark_helper import Precision, prepare_environment, setup_logger  # noqa: E402
+from llama_parity import main as parity_check  # noqa: E402
+from onnx_model import OnnxModel  # noqa: E402
+
+from onnxruntime import quantization  # noqa: E402
+
+logger = logging.getLogger("")
+
+
+def get_model_inputs(config: LlamaConfig):
+    batch_size, seq_len = 2, 8
+    input_ids = torch.randint(low=0, high=config.vocab_size, size=(batch_size, seq_len), dtype=torch.int64)
+    attn_mask = torch.randint(low=0, high=2, size=(batch_size, seq_len), dtype=torch.int64)
```

Comment:
nit: Usually we use left side padding for GPT like model. If we want to make it like real input, see example in:
https://github.com/microsoft/onnxruntime/blob/c424e42594d92daba54f264c1c7409e53529d933/onnxruntime/python/tools/transformers/models/gpt2/gpt2_helper.py#L214-L228

Reply:
This is for exporting so we can use torch.ones here.